app-mobilephone/wammu/Manifest

```diff
@@ -1,2 +1,2 @@
-DIST wammu-0.40.tar.bz2 687062 BLAKE2B 413736e40bd2a809123e176f2d00042f407c9a8bea0cc9f4a15561b7f96375c30fa1c8fc65ea8613cc7d369df13fa6841f027e69991ba323374b1dc99bf9c86d SHA512 2ed5d5cdd63b3731808c5fa8cc36caf394c4ccf714eb4ef39a25f1e7967d8ccf8b69db5aeff37ff0d1bbf7cefec51fe070a5f0b90d0cb8271f1a3d243eaf06fd
```

Comment:
You can't remove this, it's the latest stable version.

Reply:
latest stable 0.43

Reply:
Even then, removal belongs in a separate commit.

Reply:
fixed